configuration/config-txt/overclocking.md

```diff
@@ -73,14 +75,20 @@ This table describes the overvoltage settings for the various Pi models. The fir
 | Pi 3 | 1.2-1.3125V | 0 |
 | Pi Zero | 1.35V | 6 |
 
-**Specific to Pi 4B**: The `core_freq` of the Raspberry Pi 4 can change from the default if either `hdmi_enable_4kp60` or `enable_tvout` are used, due to relationship between internal clocks and the particular requirements of the requested display modes. Changing `core_freq` is not supported on the Pi 4, any change from the default will almost certainly cause a failure to boot.
+#### Specific to Pi 4B
+
+The `core_freq` of the Raspberry Pi 4 can change from the default if either `hdmi_enable_4kp60` or `enable_tvout` are used, due to relationship between internal clocks and the particular requirements of the requested display modes.
 
 |Display option | Frequency |
 |---------------|-----------|
 | Default        | 500 |     
 | enable_tvout | 360 |
 | hdmi_enable_4kp60 | 550 |
 
+Changing `core_freq` is not supported on the Pi 4, any change from the default will almost certainly cause a failure to boot.
+
+It is recommended to use `gpu_freq` rather than the individual frequency settings as this will result in best performance. 
```

Comment:
"... the individual frequency settings ... " feels a bit out of context here. Maybe "... the individual frequency settings (`h264_freq`, `isp_freq`, etc.), ..." instead?